units/host_status.go

```diff
@@ -100,7 +100,14 @@ clientsLoop:
 			return
 		}
 		if batch, ok := m.(cloud.BatchManager); ok {
+			startAt := time.Now()
 			statuses, err := batch.GetInstanceStatuses(ctx, hosts)
```

Comment:
could we instrument `GetInstanceStatuses` (and `GetInstanceStatus`) with logging as well? Assuming the API calls are what's slow, I'm curious if the problem is the call to `DescribeInstances`, `GetInstanceBlockDevices`, or `DescribeVolumes`.

Reply:
This should be handled in the [AWS client timeout ticket](https://jira.mongodb.org/browse/EVG-13763). Any AWS call could take up to 1024 seconds (over 10 minutes) so that ticket is meant to deal with the problem of restrcting arbitrarily-long AWS calls and maybe logging when they're excessively long. This ticket is just to make sure the cloud host ready job logs something of use  during its execution that's traceable to the particular job (right now it basically doesn't log anything).